src/website/app/App.js

```diff
@@ -70,6 +71,21 @@ const Nav = createStyledComponent(_Nav, styles.nav);
 const Main = createStyledComponent('main', styles.main);
 
 class App extends Component<Props> {
+  constructor(props) {
+    super(props);
+    props.history.listen((location, action) => {
```

Comment:
Can we extract this logic to a separate function?

Reply:
addressed